packages/nitro-protocol/hardhat-deploy/deploy-fvm.ts

```diff
@@ -2,24 +2,22 @@ import 'hardhat-deploy';
 import 'hardhat-deploy-ethers';
 
 import {HardhatRuntimeEnvironment} from 'hardhat/types';
-import {BigNumber} from 'ethers';
```

Comment:
Same curiosity: was this import breaking the workflow for contract verification somehow? Or just a refactor to reduce the import of effectively duplicated type info?

Reply:
This was not breaking the verification workflow, rather it is more of an optimization. Instead of using `ethers`, which is a rather large package, we are using the `hardhat-ethers` package which is already imported